samples/kubernetes/atlas-service-broker-auth.yaml

```diff
@@ -4,5 +4,27 @@ metadata:
   name: atlas-service-broker-auth
 type: Opaque
 stringData:
-  username: "wwkzdiwj@5ea0477597999053a5f9cbec"
-  password: 62bff0f6-ff2e-471d-9e36-6efd5fa24239
+  username: "username"
```

Comment:
[q] are these username/password the same as in the next secret? If so - why do we need this secret??

Reply:
one (first with username\password only) for the service-broker to get access for AOSB app, second(with JSON) for AOSB application/deployment itself